full-service/src/service/sync.rs

```diff
@@ -186,11 +212,11 @@ pub fn sync_account_next_chunk(
 
         // If no blocks were found, exit.
         if end_block_index.is_none() {
-            return Ok(());
+            return Ok((0, 0));
```

Comment:
🚧 Under Construction 🚧 

Not a huge fan of early outs in rust - I prefer to have one exit point at the end of the function. Partly because I'm also setting magic numbers here - why zero? Any reason this would change and not be 0 here?